api/auth/auth.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+require("dotenv").config();
+const express = require("express");
+const router = express.Router();
+const { google } = require('googleapis');
+
+/**
+ * Send a user a verification code for phone auth service.
+ * @async
+ * @method
+ * @param {String} phoneNumber - User's phone number
+ * @param {String} recaptchaToken - recaptchaToken generated on client side
+ * @throws {Could not send verification code} When the user cannot be send with verification code
+ */
+
+
+// route to trigger for auth
```

Comment:
Remove this line since route is already described
```suggestion
```

Reply:
> Remove this line since route is already described

@ArpitKotecha I have removed the redundant comment now. You can verify it again! Thanks